chapter_08_prettification.asciidoc

```diff
@@ -804,19 +808,23 @@ I think that looks great!
 .The lists page goes dark
 image::images/prettified-dark.png["Screenshot of lists page in dark mode. Cool."]
 
+//RITA: Whew. I was more concerned about the legibility of dark mode figures when printed, but thanks all the same.
+//RITA: On second thought, would you consider changing this to "my editor will be mad at me" or even some other silly description (freak out, self-combust, explode, etc)? I'd rather not imply that we'll cause you harm.
+
 But it's very much a matter of personal preference,
 and my editor will kill me
 if I make all the rest of my screenshots use so much ink,
 so I'm going to revert it for now.
-You feel free to keep it if you like!
+You're free to keep dark mode on if you like!
```

Comment:
why this change?

Reply:
Edit was to clarify what "it" referred to.

Reply:
aha makes sense.  thanks!